lua/octo/commands.lua

```diff
@@ -695,4 +695,31 @@ function M.issue_interactive_action(action, kind)
   M.issue_action(action, kind, value)
 end
 
+function M.command_complete(args)
+  local command_keys = vim.tbl_keys(commands)
+  local argLead, cmdLine, _ = unpack(args)
+  local parts = vim.split(cmdLine, " ")
+
+  local get_options = function(options)
+    local valid_options = {}
+    for _, option in pairs(options) do
+      if string.sub(option, 1, #argLead) == argLead then
+        table.insert(valid_options, option)
+      end
+    end
+    return valid_options
+  end
+
+  if #parts == 2 then
+    return get_options(command_keys)
+  end
+  if #parts == 3 then
```

Comment:
elseif?

Reply:
`#parts == 2` returns always. But you're right, more explicit to use `elseif`. Pushed a fix